chrome/content/zotero/zoteroPane.js

```diff
@@ -515,6 +515,18 @@ var ZoteroPane = new function()
 					this.highlightTimer = null;
 				}
 				ZoteroPane_Local.collectionsView.setHighlightedRows();
+				return;
+			} else if (event.keyCode == event.DOM_VK_BACK_QUOTE) {
```

Comment:
Can make it configurable, but I don't have a particular objection to backtick. (For what it's worth, Thunderbird uses 'M' (**M**ark?), and we do crib their number-keys approach to tagging.)